jac-uk/admin
# Copy exercise progress between environments

## What's included?
If progress property exists in data when creating an exercise (ie. when an exercise is being created from copy/clipboard) then the progress will be copied across, otherwise (ie. when creating from scratch) this is created as previously.

## Who should test?
✅ Product owner
✅ Developers
✅ UTG

## How to test?
- Create an exercise normally.
- Observe that when starting all application sections do NOT have the [DONE] tag and the exercise cannot be reviewed/approved
- Create an exercise by copying an exercise.
- Observe that when starting all application sections do have the [DONE] tag and the exercise can be reviewed/approve immediately.
 
## Risk - how likely is this to impact other areas?
🟢 No risk - this is a self-contained piece of work

## Additional context
...coming soon...

---
PREVIEW:DEVELOP
_can be OFF, DEVELOP or STAGING_
